src/libutil/config.cc

```diff
@@ -70,10 +70,17 @@ void AbstractConfig::reapplyUnknownSettings()
         set(s.first, s.second);
 }
 
+static bool regular(const Config::SettingData & sd)
```

Comment:
```suggestion
// Whether we should process the option. Excludes aliases, which are handled elsewhere, and disabled features.
static bool applicable(const Config::SettingData & sd)
```

I think this name is a bit more meaningful. Where are aliases handled?

Reply:
Not sure, but I didn't that.